Extensions/Physics2Behavior/tests/physics2runtimebehavior.spec.js

```diff
@@ -318,4 +857,104 @@ describe('Physics2RuntimeBehavior', () => {
       expect(behavior.contactsEndedThisFrame.length).to.be(0);
     });
   });
+
+  describe('Behavior interaction with other objects behaviors and extensions.', () => {
+    let runtimeGame;
+    let runtimeScene;
+    beforeEach(() => {
+      [runtimeGame, runtimeScene] = createGameWithSceneWithPhysics2SharedData();
+    });
+    it('BehaviorTest should have access to current started and ended contacts in postEvent.', () => {
```

Comment:
Having the asserts during the postEvent of another behavior or after the rendering is likely the same unless you want to test a behavior that run before the Physics one.

But even in this case, as there is no more side effect in the postEvent of Physics, a less strong tests might be ok for the sake of tests readability. I guess a comment to warn to avoid putting side effect in the postEvent should be enough.

Note that `renderAndStep` actually steps and renders.

Reply:
Mmmh, do you think this test is still relevant then? Since we already assert quite a lot after renderAndStep in other tests?